arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,495 @@
+                    "type": "string",
+                    "description": "Gets or sets the name returned from the server."
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Gets or sets the type returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportExportOperationStatusResponseProperties": {
+            "properties": {
+                "requestType": {
+                    "type": "string",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "requestId": {
+                    "type": "string",
+                    "format": "uuid",
+                    "description": "Gets or sets the request type of the operation."
+                },
+                "serverName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Server."
+                },
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL Database."
+                },
+                "status": {
+                    "type": "string",
+                    "description": "Gets or sets the status message returned from the server."
+                },
+                "lastModifiedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation status last modified time."
+                },
+                "queuedTime": {
+                    "type": "string",
+                    "description": "Gets or sets the operation queued time."
+                },
+                "blobUri": {
+                    "type": "string",
+                    "description": "Gets or sets the blob uri."
+                },
+                "errorMessage": {
+                    "type": "string",
+                    "description": "Gets or sets the error message returned from the server."
+                }
+            },
+            "description": "Response for Azure Sql Import/Export Status operation."
+        },
+        "ImportRequestParameters": {
+            "properties": {
+                "databaseName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the Azure SQL database to import."
+                },
+                "edition": {
+                    "type": "string",
+                    "description": "Gets or sets the edition for the Azure Sql Database being created."
+                },
+                "serviceObjectiveName": {
+                    "type": "string",
+                    "description": "Gets or sets the name of the service objective to assign to the Azure SQL Database."
+                },
+                "maxSizeBytes": {
```

Comment:
There are constraints on the max size of the database, but they are determined by the Edition+Service Objective (Pricing Tier).

I believe the backend service expects this as a string.  In future API versions we can change it to a long.